web/index.html

```diff
@@ -0,0 +1,68 @@
+<html lang="en">
+<head>
+    <meta charset="utf-8">
+    <title>BCU Labs</title>
+    <meta name="viewport"
+          content="initial-scale=1,maximum-scale=1,user-scalable=no">
+    <link href="https://api.mapbox.com/mapbox-gl-js/v3.1.2/mapbox-gl.css"
+          rel="stylesheet">
+    <style>
+        body {
+            margin: 0;
+            padding: 0;
+        }
+
+        #map {
+            position: absolute;
+            top: 0;
+            bottom: 0;
+            width: 100%;
+        }
+    </style>
+</head>
+<body>
+<h1>BCU Data Labs</h1>
+<div id="map"></div>
+<script src="https://api.mapbox.com/mapbox-gl-js/v3.1.2/mapbox-gl.js"></script>
+<script>
+    const baseWidth = 1.4;
+    const baseZoom = 12
+
+    mapboxgl.accessToken = 'pk.eyJ1IjoidHNjaGllZ2dtIiwiYSI6ImNrZHoyb25iYjMxMWQzM2p6eHlibHJkanIifQ.wj-SisFvNPgakxBy_1ZnHw';
+    const map = new mapboxgl.Map({
+        container: 'map', // container ID
+        center: [-71.057083, 42.361145], // Boston starting position [lng, lat]
+        style: "mapbox://styles/tschieggm/cke3eofps022p19mqmvfc8n4q",
+        zoom: baseZoom // starting zoom
+    });
+
+    map.on('load', function () {
+        map.addSource('lts', {
+            type: 'geojson',
+            data: 'plots/Boston_LTS_lonboard.json'
```

Comment:
is it possible to programmatically choose which city to plot?

Reply:
Yes, in the future. This is a proof of concept with hardcoded values for Boston.